File: java/iroha_android/src/main/java/org/hyperledger/iroha/android/offline/attestation/SafetyDetectException.java
package org.hyperledger.iroha.android.offline.attestation;

/** Runtime exception thrown when Safety Detect attestation fails. */
public final class SafetyDetectException extends RuntimeException {

  private static final long serialVersionUID = 1L;

  public SafetyDetectException(final String message) {
    super(message);
  }

  public SafetyDetectException(final String message, final Throwable cause) {
    super(message, cause);
  }
}

File: kotlin/offline-wallet-android/src/main/java/org/hyperledger/iroha/sdk/offline/attestation/SafetyDetectException.kt
package org.hyperledger.iroha.sdk.offline.attestation

/** Runtime exception thrown when Safety Detect attestation fails. */
class SafetyDetectException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable?) : super(message, cause)

    companion object {
        private const val serialVersionUID: Long = 1L
    }
}
